### ISSUE TITLE
aria warnings in lighthouse for input element

### ISSUE THREAD

**Issue by whicks1** [corejavascript/typeahead.js#195]
Using 1.2.1 release a chrome developer tools/lighthouse audit shows failure:

 "[aria-*] attributes do not match their role."

**Comment by whicks1** [comment]
Followup, the input attempts to point to the listbox with aria-owns, but the generated listbox doesn't enforce/add and ID and so it fails an additional test here
